code/datums/gamemode/factions/bloodcult/bloodcult_rituals.dm

```diff
@@ -358,14 +225,430 @@ var/global/list/cult_altars = list()       // List of cult altars in the world.
+
+/datum/bloodcult_ritual/blind_cameras/key_found(var/extra)
+	target_cameras--
+	if(target_cameras <= 0)
+		return TRUE
+	return FALSE
+
+////////////////////////////////////////////////////////////////
+
+/datum/bloodcult_ritual/confuse_crew
+	name = "Confuse Crew"
+	desc = "confusion runes and talismans...<br>bring their nightmares to life..."
+
+	ritual_type = "confusion"
+	difficulty = "medium"
+	personal = TRUE
+	reward_achiever = 200
+	reward_faction = 2
+
+	keys = list(
+		"confusion_carbon",
+		"confusion_papered",
+		)
+
+/datum/bloodcult_ritual/confuse_crew/key_found(var/mob/living/extra)
+	if (!extra.client)
+		return FALSE
+	return TRUE
+
+////////////////////////HARM CREW MEMBERS/////////////////////////////
+
+/datum/bloodcult_ritual/harm_crew
+	name = "Harm Crew"
+	desc = "wield cult weaponry...<br>spill their blood...<br>sear their skin..."
+
+	ritual_type = "harm"
+	difficulty = "medium"
+	personal = TRUE
+	reward_achiever = 200
+	reward_faction = 2
+
+	keys = list(
+		"attack_tome",
+		"attack_cultblade",
+		"attack_blooddagger",
+		"attack_construct",
+		"attack_shade",
+		"attack_ritualknife",
+		)
+
+	var/targets = 3
+	var/list/hit_targets = list()
+
+/datum/bloodcult_ritual/harm_crew/init_ritual()
+	hit_targets = list()
+
+/datum/bloodcult_ritual/harm_crew/update_desc()
+	desc = "wield cult weaponry...<br>spill their blood...<br>sear their skin...<br>at least [targets - hit_targets] different individuals..."
+
+/datum/bloodcult_ritual/harm_crew/key_found(var/mob/living/L)
+	if (iscultist(L))
+		return FALSE
+	if (L in hit_targets)
+		return FALSE
+	hit_targets += L
+	if(hit_targets >= targets)
+		return TRUE
+	return FALSE
+
+////////////////////////SACRIFICE/////////////////////////////
+
+/datum/bloodcult_ritual/sacrifice_mouse
+	name = "Sacrifice Mouse"
+	desc = "a rodent...<br>an altar...<br>and a proper blade..."
+
+	ritual_type = "sacrifice"
+	difficulty = "easy"
+	personal = TRUE
+	reward_achiever = 200
+	reward_faction = 2
+
+	keys = list("altar_sacrifice_animal")
+
+/datum/bloodcult_ritual/sacrifice_mouse/key_found(var/mob/living/simple_animal/mouse/extra)
+	if(istype(extra))
+		return TRUE
+	return FALSE
+
+
+//////////////////////////////////////////////////////////////
+
+/datum/bloodcult_ritual/sacrifice_monkey
+	name = "Sacrifice Monkey"
+	desc = "a simian...<br>an altar...<br>and a proper blade..."
+
+	ritual_type = "sacrifice"
+	difficulty = "easy"
+	personal = TRUE
+	reward_achiever = 200
+	reward_faction = 2
+
+	keys = list("altar_sacrifice_monkey")
+
+/datum/bloodcult_ritual/sacrifice_monkey/key_found(var/extra)
+	return TRUE
+
+
+////////////////////////ALTAR/////////////////////////////////
+
+/datum/bloodcult_ritual/altar
+	name = "Prepare Altar"
+	desc = "raise an altar...<br>add proper paraphernalia around...<br>then plant a ritual knife on top..."
+
+	ritual_type = "altar"
+	difficulty = "easy"
+	personal = TRUE
+	reward_achiever = 200
+	reward_faction = 2
```

Comment:
This should definitely be a larger reward than sacrificing a monkey or mouse.

Reply:
"easy" rituals usually involve no direct interactions with the crew, and the cultists will generate some additional devotion regardless from the mere acts of raising the altar, spawning blood candles, etc. Rituals are just a cherry on top that may inspire cultists to do something cult-like other than the basic "convert people/make weapons". I don't think we need to overbalance ritual values, but we'll see what the players think.

I sure hope people aren't gonna fill the wiki with a list of everything that grants devotion and how much so players can optimize the fun out of the game.